File: docs/examples/sdk/typescript/quickstart.ts
import { Configuration, DefaultApi } from '../../../sdk/typescript'

async function main() {
  const env: any = (globalThis as any).process?.env || {}
  const api = new DefaultApi(
    new Configuration({
      basePath: env.API_URL || 'http://localhost:8000/api/v1',
      headers: {
        Authorization: `Bearer ${env.API_TOKEN || ''}`,
        'X-API-Key': env.API_KEY || '',
      },
    })
  )

  const res = await api.healthzGet()
  // eslint-disable-next-line no-console
  console.log(res)
}

main().catch((e) => {
  // eslint-disable-next-line no-console
  console.error(e)
  ;(globalThis as any).process?.exit?.(1)
})

File: docs/examples/sdk/python/quickstart.py
from pathlib import Path
import os
import sys

# Add generated SDK to path (adjust if installed as a package)
sdk_path = Path(__file__).resolve().parents[3] / 'docs' / 'sdk' / 'python'
sys.path.append(str(sdk_path))

from blockchain_forensics_sdk import ApiClient, Configuration, DefaultApi  # type: ignore


def main() -> None:
    cfg = Configuration()
    cfg.host = os.getenv('API_URL', 'http://localhost:8000/api/v1')
    if api_key := os.getenv('API_KEY'):
        cfg.api_key['X-API-Key'] = api_key
    if token := os.getenv('API_TOKEN'):
        cfg.access_token = token

    with ApiClient(cfg) as client:
        api = DefaultApi(client)
        res = api.api_v1_system_health_get()
        print(res)


if __name__ == '__main__':
    main()
